cluster-up/cluster/ocp-4.3/README.md

```diff
@@ -0,0 +1,45 @@
+# OKD 4.3 in ephemeral containers
+
+Provides a pre-deployed OKD with version 4.3 purely in docker
+containers with libvirt. The provided VMs are completely ephemeral and are
+recreated on every cluster restart. The KubeVirt containers are built on the
+local machine and are then pushed to a registry which is exposed at
+`localhost:5000`.
+
+## Bringing the cluster up
+
+```bash
+export KUBEVIRT_PROVIDER=okd-4.3
+export KUBEVIRT_NUM_NODES=3 # master + two workers
```

Comment:
I thought we would only need one worker?

Reply:
This is just an example of how you can start it up with more workers if needed.